src/routes.js

```diff
@@ -0,0 +1,12 @@
+const usersHandlers = require('./handlers/users')
+
+const router = require('express').Router()
+
+// user routes
+router.get('/users', usersHandlers.get)
```

Comment:
We can go
```js
router.route('/users')
   .get(usersHandlers.getAll)
   .post(userHandlers.create)

router.route('/users/:id')
   .get(getById)
   .put
..
```
Other Thing is that they are called controllers, not handlers, you should have a folder called controllers